examples/multiple-certs/main.tf

```diff
@@ -42,143 +42,228 @@ variable "network_name" {
   default = "tf-lb-https-multi-cert"
 }
 
+variable "service_account" {
```

Comment:
Same

Reply:
@ingwarr, thank you for review! [Fixed](https://github.com/terraform-google-modules/terraform-google-lb-http/compare/3863b4b4031bee488cf54f700867f67fea71b878..faaadf20455456bce10b8910bd867134b3a70dbe)